src/test/java/com/bookstore/uitests/pageobjectmodels/UsersPage.java

```diff
@@ -0,0 +1,93 @@
+package com.bookstore.uitests.pageobjectmodels;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class UsersPage {
+    private WebDriver driver;
+
+    public UsersPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = "users-btn")
+    private WebElement usersButton;
+
+    @FindBy(id = "user-id")
+    private WebElement userIdInput;
+
+    @FindBy(id = "search-btn")
+    private WebElement searchButton;
+
+    @FindBy(id = "create-btn")
+    private WebElement createButton;
+
+    @FindBy(id = "edit-btn")
+    private WebElement editButton;
+
+    @FindBy(id = "delete-btn")
+    private WebElement deleteButton;
+
+    @FindBy(id = "search-results")
+    private WebElement searchResultsTable;
+
+    @FindBy(id = "user-id-pop-up")
+    private WebElement userIdPopUp;
+
+    @FindBy(id = "close-btn")
+    private WebElement closePopUpButton;
+
+    @FindBy(xpath = "/html/body/div[3]/table/tbody/tr[2]/td")
```

Comment:
as you have developed UI - you can specify the id attribute in the source code for this particular element and refer to it here